### ISSUE TITLE
add the ability to select/configure which IP(s) the l1 binds to

### ISSUE THREAD

**Issue by gruns** [filecoin-saturn/L1-node#77]
many servers have multiple IPs, and its super handy to be able to select which IP(s) should be bound to

kudos to @jcace for the suggestion

**Comment by typoworx-de** [comment]
I'll go for this to! I think binding multiple IPs doesn't make sense at all for this type of service?!

**Comment by gruns** [comment]
yep! binding to multiple IPs indeed makes little sense. no need to support that for now

the key is to let l1 nodes select which IP (IPv4 for now, IPv6 soon) to bind to. many servers come with multiple IPs and the l1 node operator should be able to select which IP to bind to
